exes/mqtt-broadcaster/inc/mqtt_broadcaster.hpp

```diff
@@ -654,6 +763,8 @@ class mqtt_broadcaster {
 
   asio::cancellation_signal cancel_signal_;
 
+  std::vector<scada_signal> scada_signals;
```

Comment:
this could be `std::vector<ipc::any_signal>` given the config struct

Reply:
How would I access the name and the signal in the same iteration when searching for a signal?